### PR TITLE
Show Qiskit depreciation logs in report

### LINKED ISSUE #81: Print Qiskit depreciation warning in submission report
### What is the expected enhancement?
Print depreciation warning in the tests check report in the submission issue

---

Epic #9

### PR DESCRIPTION
**Pull request by mickahell**
### Summary
Adding the Qiskit depreciation logs in report issue

### Details and comments
See #81 

---
Closes #81 
Epic #9

### CODE REVIEW

**Comment by IceKhan13** [comment]
😄 

Screen Shot 2021-11-04 at 10 02 21 AM

**Comment by mickahell** [comment]
> 
> 
> 😄 just to be 100% sure you merged it
> 
> Screen Shot 2021-11-04 at 10 02 21 AM

I don't know why my fork always keeping those commit ^^'

**Comment by mickahell** [comment]
So @IceKhan13 what do you thing of that :)

File: ecosystem/models/execution_summary.py [hunk 1]
```diff
@@ -27,6 +27,14 @@ def get_warning_logs(self) -> List[str]:
         """Return warning messages."""
         return [log for log in self.logs if "warn" in log.lower()]
 
+    def get_qiskit_depreciation_logs(self) -> List[str]:
+        """Return qiskit depreciation messages."""
+        return [
```
Comment (IceKhan13) [inline ecosystem/models/execution_summary.py:32]: Duplicate code of 
https://github.com/qiskit-community/ecosystem/blob/fe9730da03648bdfebaad00dc151f669f2072059/ecosystem/models/execution_summary.py#L49-L53

let's reuse chunks of code in here

File: ecosystem/runners/runner.py [hunk 2]
```diff
@@ -141,8 +141,10 @@ def run(self) -> Tuple[str, List[CommandExecutionSummary]]:
             logs_error = []
             logs_fail = []
             for element in executive_summary:
+                logs_depreciation += element.get_qiskit_depreciation_logs()
                 logs_error += element.get_error_logs()
                 logs_fail += element.get_fail_logs()
+            set_actions_output([("DEPRECIATION", "\n".join(logs_depreciation))])
```
Comment (IceKhan13) [inline ecosystem/runners/runner.py:147]: we can use only one call

```python
set_actions_output([
    ("DEPRECIATION", "\n".join(logs_depreciation)),
    ("ERROR", "\n".join(logs_error)),
    ("FAIL", "\n".join(logs_fail))
])
```

**Review by IceKhan13 — CHANGES_REQUESTED** [review]
Few things, but overall it suppose to work :)

File: ecosystem/models/execution_summary.py (hunk 1 above)
Comment (mickahell) [reply ecosystem/models/execution_summary.py:32]: Indeed

**Review by IceKhan13 — APPROVED** [review]
LGTM!

**Pull request merged**